Source/WebCore/css/CSSCounterStyleDescriptors.cpp

```diff
@@ -67,51 +67,78 @@ static String symbolToString(const CSSValue* value)
     return primitiveValue.stringValue();
 }
 
-static CSSCounterStyleDescriptors::Pad translatePadFromStyleProperties(const StyleProperties& properties)
+static CSSCounterStyleDescriptors::AdditiveSymbols translateAdditiveSymbolsFromStyleProperties(const StyleProperties& properties)
 {
-    auto pad = properties.getPropertyCSSValue(CSSPropertyPad).get();
-    if (!pad || !is<CSSValueList>(pad))
-        return CSSCounterStyleDescriptors::Pad();
+    auto additiveSymbols = properties.getPropertyCSSValue(CSSPropertyAdditiveSymbols);
+    if (!additiveSymbols)
+        return { };
+    CSSValueList* list;
+    if (!(list = downcast<CSSValueList>(additiveSymbols.get())))
+        return { };
```

Comment:
```suggestion
    auto additiveSymbols = properties.getPropertyCSSValue(CSSPropertyAdditiveSymbols);
    auto* list = dynamicDowncast<CSSValueList>(additiveSymbols.get());
    if (!list)
        return { };
```

(Same above with ranges)

Reply:
this doesn't seem right, we should check for `!additiveSymbols`, but I can do the `(!list)` check on a separated line if that's what you mean

Reply:
Missing properties happen. A property with a value of the wrong type can’t happen. The two checks aren’t equivalent. One is needed, the other is dead code and we’d be better off with an assert.